Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    

    <meta name="description" content="This is a sample webpage." />
    <meta name="keywords" content="HTML, Meta tags, Web development" />
    <meta name="author" content="vaibhav shukla" />
    
    <title>Webpage with Meta Tags</title>
</head>
<body>

    <h1>Welcome to the meta tags</h1>

    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aperiam ipsam itaque suscipit sapiente ad fuga nostrum alias dolore similique, minima, natus cum! Sequi modi et repudiandae temporibus assumenda, eligendi veritatis?</p>

    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sapiente, fuga?</p>

</body>
</html>